test/factories/editions.rb

```diff
@@ -157,24 +157,28 @@
     end
 
     trait(:unpublished) do
+      state { :unpublished }
       after(:create) do |edition|
         edition.unpublishing = build(:unpublishing, edition:)
       end
```

Comment:
Nope, just thoughtless copy pasta

Reply:
As mentioned, this proposed change broke some tests. I don't exactly know why, but essentially when the after create hook is used the test/unit/app/services/asset_manager/attachment_updater_test.rb:216 fails because the edition doesn't seem to have a thumbnail any longer. It also affects other tests around attachments and assets.

Reply:
This change exposed some incorrect tests which I have resolved.